packages/clerk_flutter/lib/src/widgets/ui/clerk_text_form_field.dart

```diff
@@ -65,15 +65,18 @@ class ClerkTextFormField extends StatelessWidget {
           height: 32.0,
           child: DecoratedBox(
             decoration: inputBoxBorderDecoration,
-            child: _TextField(
-              optional: isOptional,
-              obscureText: obscureText,
-              onChanged: onChanged,
-              onSubmit: onSubmit,
-              initial: initial,
-              onObscure: onObscure,
-              validator: validator,
-              autofocus: autofocus,
+            child: Material(
```

Comment:
Material should have same shape as parent DecoratedBox or touch feedback ripples will go outside of corner radius.